02-express-tutorial/02-http-app.js

```diff
@@ -0,0 +1,44 @@
+const http = require('http')
+const { readFileSync } = require('fs')
+
+const homePage = readFileSync('./navbar-app/index.html')
+const homeStyles = readFileSync('./navbar-app/styles.css')
+const homeImage = readFileSync('./navbar-app/logo.svg')
+const homeLogic = readFileSync('./navbar-app/browser-app.js')
```

Comment:
Extra challenge question: If we had 10,000 files instead of 4, what problems would we encounter with this design? How could we change the code to overcome some of those problems?